api-tester/src/main/java/com/revenuecat/apitester/java/PurchasesAPI.java

```diff
@@ -73,8 +73,8 @@ static void check(final Purchases purchases,
 
         purchases.syncPurchases();
         purchases.getOfferings(receiveOfferingsListener);
-        purchases.getSubscriptionSkus(skus, skusResponseListener);
-        purchases.getNonSubscriptionSkus(skus, skusResponseListener);
+        purchases.getSubscriptionProducts(productIds, skusResponseListener);
```

Comment:
definitely in favor or replacing sku references!

though really this should've been `SkuDetails` in the first place, `sku` more directly maps to `productId`

Reply:
Also rename `skusResponseListener` to `productResponseListener`, then?